install/listmonk-install.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: bvdberg01
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  curl \
+  sudo \
+  postgresql
+msg_ok "Installed Dependencies"
+
+msg_info "Setting up PostgreSQL"
+DB_NAME=listmonk
+DB_USER=listmonk
+DB_PASS=$(openssl rand -base64 18 | tr -dc 'a-zA-Z0-9' | cut -c1-13)
+$STD sudo -u postgres psql -c "CREATE ROLE $DB_USER WITH LOGIN PASSWORD '$DB_PASS';"
+$STD sudo -u postgres psql -c "CREATE DATABASE $DB_NAME WITH OWNER $DB_USER TEMPLATE template0;"
+{
+echo "listmonk-Credentials"
+echo -e "listmonk Database User: \e[32m$DB_USER\e[0m"
+echo -e "listmonk Database Password: \e[32m$DB_PASS\e[0m"
+echo -e "listmonk Database Name: \e[32m$DB_NAME\e[0m"
+} >> ~/listmonk.creds
+msg_ok "Set up PostgreSQL"
+
+msg_info "Installing ${APPLICATION}"
```

Comment:
```suggestion
msg_info "Installing listmonk"
```
Maybe we do this improvement later, but for now keep same as other scripts